test/jdk/tools/jpackage/macosx/base/SigningCheck.java

```diff
@@ -89,4 +90,18 @@ private static void validateCertificateTrust(String name) {
                 .apply(result.stream());
     }
 
+    public static void isXcodeDevToolsInstalled() {
+        ProcessBuilder pb = new ProcessBuilder("/usr/bin/xcrun", "--help");
+        try {
+            Process p = pb.start();
+            int code = p.waitFor();
+            if (code != 0) {
+                TKit.throwSkippedException("Missing Xcode with command line " +
+                        "developer tools");
+            }
+        } catch (IOException | InterruptedException ex) {
+            throw new RuntimeException(ex);
+        }
```

Comment:
Can be reduced to:

```
public static void isXcodeDevToolsInstalled() {
  if (Executor.of("/usr/bin/xcrun", "--help").execute().getExitCode() != 0) {
    TKit.throwSkippedException("Missing Xcode with command line developer tools");
  }
}
```

Also will add a log record to help evaluate potential test failure.

Reply:
Fixed, but I am using `executeWithoutExitCodeCheck()` instead of  `execute()`, `execute()` throws exception if command returns non-zero value.